.github/workflows/release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           - ./build.sh feature
           - ./build.sh develop
         platform:
-          - linux/amd64,linux/arm64
+          - linux/amd64
```

Comment:
If https://github.com/netbox-community/netbox-docker/pull/1303 works out, I'd rather wait for that, than have no `arm64` images meanwhile.